banners/desktop/banner_var.ts

```diff
@@ -75,6 +74,5 @@ app.provide( 'formItems', createFormItems( translator, currencyFormatter.euroAmo
 app.provide( 'formActions', createFormActions( page.getTracking(), impressionCount, { des: '1' } ) );
 app.provide( 'tracker', tracker );
 app.provide( 'dailyDonorAverage', dailyDonorAverage );
```

Comment:
This is not longer being used, you can delete it. You can also delete the `dailyDonorAverage` const and the `LinearDailyDonorAverage` and `IntegerDe` imports.

Reply:
Done.